tools/modulelint.py

```diff
@@ -46,8 +46,30 @@ def setUp(self):
     def testDockerFromBaseruntime(self):
         self.assertTrue(self.dp.check_baseruntime())
 
-    def testDockerRunMicrodnf(self):
-        self.assertTrue(self.dp.check_microdnf())
+    def testDockerNodocs(self):
+        self.start()
+        installed_pkgs = self.run("rpm -qa --qf '%{{NAME}}\n'", ignore_status=True).stdout
+        # This returns a list of packages defined in config.yaml for testing
+        # e.g. ["bash", "rpm", "memcached"] in case of memcached
+        pkgs = self.backend.getPackageList()
+        for pkg in installed_pkgs.split('\n'):
+            if pkg in pkgs:
+                all_docs = self.run("rpm -qd %s" % pkg).stdout
+                for doc in all_docs.strip().split('\n'):
+                    self.assertNotEqual(0, self.run("test -e %s" % doc, ignore_status=True).exit_status)
+
+    def testDockerCleanAll(self):
+        self.start()
+        pkg_mgr = "yum"
+        # Detect distro in image
+        distro = self.run("cat /etc/os-release").stdout
```

Comment:
this is custom file which might not exist, original file is /usr/lib/os-release which can be modified by /etc one... and just parsing those files is really bad idea ;)

Reply:
@ignatenkobrain What do you prefer?

Reply:
I vote for  /etc/os-release :)
I guess that /etc/os-release is the best way for quick linux check. This file is usually not changed intentionally.

if you want to be very strict look at following tool that check it more precisely:
https://unix.stackexchange.com/questions/6345/how-can-i-get-distribution-name-and-version-number-in-a-simple-shell-script